pep-0589.rst

```diff
@@ -274,6 +274,17 @@ The TypedDict ``XYZ`` has three items: ``x`` (type ``int``), ``y``
 A TypedDict cannot inherit from both a TypedDict type and a
 non-TypedDict base class.
 
+A TypedDict field cannot be overwritten by inheritance. Example::
+
+      class X(TypedDict):
+         x: int
+
+      class Y(TypedDict):
+         y: str
+
+      class XYZ(X, Y):
+         y: bool  # Type check error: Cannot overwrite TypedDict field "y"
+
```

Comment:
I suspect we need to say something more. The phrase you added doesn't refer to *multiple* inheritance but the example uses that. I suspect we need to provide at least two examples: one where it is shown that single inheritance cannot change a field's type in the subclass, one where it is shown that multiple inheritance does not allow conflicting types for the same field name. Your current example combines both but does not make it clear that each of the simpler constructs is also disallowed.